knowlg-app/src/app/player/player.page.ts

```diff
@@ -27,7 +27,7 @@ export class PlayerPage implements OnInit {
     });
   }
   playerEvent(event) {
-    const eventType = _.get(event, 'edata.type');
+    const eventType = _.get(event, 'detail.edata.type');
```

Comment:
Why is this detail is added now?

Reply:
As we implemented Web component the event details comes as part of **detail** , FYI 
<img width="1244" alt="Screenshot 2022-11-18 at 11 24 02 AM" src="https://user-images.githubusercontent.com/56960091/202630916-ea946981-e7dc-4089-982c-85ec45eb35a0.png">